src/stats.jl

```diff
@@ -191,30 +191,39 @@ function hpd(chn::Chains; alpha::Real=0.05, kwargs...)
 end
 
 """
-    quantile(chains[; q = [0.025, 0.25, 0.5, 0.75, 0.975], append_chains = true, kwargs...])
+    quantile(chains, q = [0.025, 0.25, 0.5, 0.75, 0.975]; alpha, beta, kwargs...)
+    quantile(chains; q, alpha, beta, kwargs...)
```

Comment:
These changes are not required for this PR, are they?
```suggestion
    quantile(chains::Chains; q::AbstractVector = [0.025, 0.25, 0.5, 0.75, 0.975], kwargs...)
```